### PR TITLE
Adding logging to ephemeris generation.

### PR DESCRIPTION
**Pull request by astronomerritt**
This is the setup for #525. I've added verbose logging to `create_ephemeris` in `simulation_driver.py`, and also added a few logging info messages in that function. 

If you want to add more log messages:

```
verboselog("This prints a message to the info log if verbose == True.")
args.pplogger.info("This forces a log message to the info log.")
args.pplogger.error("This prints a message to the error log.")
```

Adding logging to _another_ function is as simple as making sure args is passed to that function, then to enable verbose logging, add at the top of the function:

`verboselog = args.pplogger.info if args.verbose else lambda *a, **k: None`

You can see how I've done this at the top of `create_ephemeris`.

## Review Checklist for Source Code Changes

- [x] Does pip install still work?
- [ ] Have you written a unit test for any new functions?
- [x] Do all the units tests run successfully?
- [x] Does Sorcha run successfully on a test set of input files/databases?
- [x] Have you used black on the files you have updated to confirm python programming style guide enforcement?

### CODE REVIEW

File: src/sorcha/ephemeris/simulation_driver.py
Comment (mschwamb) [inline src/sorcha/ephemeris/simulation_driver.py:103]: Can we be more specific here ASSIST+REBOUND simulations

**Comment by mschwamb** [comment]
Just one minor change. Then it can be merged.

**Comment by mschwamb** [comment]
I'll approve if you can make that small change @astronomerritt

**Review by mschwamb — APPROVED** [review]

File: src/sorcha/ephemeris/simulation_driver.py
Comment (astronomerritt) [reply src/sorcha/ephemeris/simulation_driver.py]: Done, just waiting for the tests to pass then I'll merge.

**Pull request merged**
